src/app/src/App.js

```diff
@@ -2,26 +2,57 @@ import { Box, Flex } from '@chakra-ui/react';
 import { BrowserRouter, Navigate, Routes, Route } from 'react-router-dom';
 
 import './App.css';
+import Login from './pages/Login';
 import Welcome from './pages/Welcome';
 import Draw from './pages/Draw';
 import Map from './components/Map';
 import Sidebar from './components/Sidebar';
 
+import PrivateRoute from './components/PrivateRoute';
+
 function App() {
     return (
         <BrowserRouter>
             <Flex>
                 <Routes>
-                    <Route path='/draw' element={<Sidebar />} />
+                    <Route
+                        path='/draw'
```

Comment:
Interestingly, `/draw` does redirect me, but `/draw/` (which I usually use) does not.

Reply:
This is concerning: apparently, this is the request axios is making, which obviously fails because there are no endpoints exposed under `/draw/`.
```
GET http://localhost:4545/draw/api/auth/login/ 404 (Not Found)
```

I'll troubleshoot on Monday, but let me know if anything comes to mind immediately.

Reply:
Actually, c97adef49821ccc06c239e591a3ed9cb9fb4b4ca seemed to do it.